src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -61,6 +61,48 @@
  * <pre>
  * &lt;module name=&quot;NoWhitespaceBefore&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * int foo;
+ * foo ++; // violation, whitespace before '++' is not allowed
+ * foo++; // OK
+ * for (int i = 0 ; i &lt; 5; i++) {}  // violation
+ *            // ^ whitespace before ';' is not allowed
+ * for (int i = 0; i &lt; 5; i++) {} // OK
+ * int[][] array = { { 1, 2, 3 }
+ *                 , { 4, 5, 6 }    // violation,
+ *                 , { 7, 8, 9 } }; // violation, whitespace before ',' is not allowed
```

Comment:
two equal violations.

please make it as:
```
 *                 ,{ 4, 5, 6 }    // OK,
 *                 , { 7, 8, 9 } }; // violation, whitespace before ',' is not allowed
```

Reply:
```
 *                 ,{ 4, 5, 6 }    // OK,
 *                 , { 7, 8, 9 } }; // violation, whitespace before ',' is not allowed
```
Both the lines have violation, so how can I say OK on first line.. or you are saying to make this  array a 2x3 matrix, this way there will be single violation on the second line.

Reply:
as violation is the same please remove extra line, we do not need to show same violations to user.